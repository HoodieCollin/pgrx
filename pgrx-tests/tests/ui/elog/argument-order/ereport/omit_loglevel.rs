use pgrx::prelude::*;

fn main() {}

#[pg_extern]
fn ereport_loglevel_is_required() {
    ereport!(
        errcode = PgSqlErrorCode::ERRCODE_SUCCESSFUL_COMPLETION;
        message = "...";
    );
}
