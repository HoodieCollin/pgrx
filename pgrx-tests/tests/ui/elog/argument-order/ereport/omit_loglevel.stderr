error: The alternative invocation style requires `loglevel` argument to be specified first.
  --> tests/ui/elog/argument-order/ereport/omit_loglevel.rs:7:5
   |
7  | /     ereport!(
8  | |         errcode = PgSqlErrorCode::ERRCODE_SUCCESSFUL_COMPLETION;
9  | |         message = "...";
10 | |     );
   | |_____^
   |
   = note: this error originates in the macro `ereport` (in Nightly builds, run with -Z macro-backtrace for more info)
